app/templates/jinja2/missions.html

```diff
@@ -32,7 +32,7 @@
 </header>
 
 <main class="ui-layout" id="landing">
-  <div class="mission_details">
+  <div class="full-height">
```

Comment:
Why?

Reply:
I made a reusable class that essentially we can use in a bunch of places that will make the page full height.